src/constants/colors/index.ts

```diff
@@ -76,6 +76,7 @@ export const colorAliases = {
   primaryTint1: brandColors.purple85,
   primaryTint2: brandColors.purple70,
   primaryTint3: brandColors.purple30,
+  primaryTint4: brandColors.purple15,
```

Comment:
I suppose this will be removed anyway, so it does not really matter. However the PocketDerm review app currently depends on it.